extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -2209,4 +2209,20 @@ public static function onArticleCommentGetSquidURLs( $title, &$urls ) {
 		return true;
 	}
 
+	/**
+	 * @desc Fix User_talk links for profile page diff on wall enabled wikis
```

Comment:
Should this comment be more generic? I.e. "Convert talk page links to wall page links for wall enabled wikis". TBH I'm not 100% sure that's what this function is doing (I'm not super familiar with this code), but it seems like that's what it should be doing. If the talk page is requested for a user on a wall-enabled wikia, regardless of the circumstances, we should return the wall page. 

Also, `@desc` is unnecessary here. Text without tags defaults to the description ([docs](http://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.pkg.html)).